docs/concepts.rst

```diff
@@ -316,17 +316,44 @@ file as long as the ``major`` version matches.
     def message(greeting, subject):
         return f'{greeting} {subject}!!!'.upper()
 
-Be aware that Bionic can't detect every change that can affect your code's
-behavior.  It only looks at the code of the decorated function itself; if you
-change any other function or library that your decorated function depends on,
-Bionic won't notice.  Similarly, if your function is wrapped by a non-Bionic
-decorator, Bionic won't detect any code changes in that function at all.
-That's why this mode only provides a warning, rather than automatically
-invalidating the cache for you: to keep you in the habit of thinking carefully
-about versioning.
 
-However, if you do want to completely automate the versioning process, you
-can set Bionic to a "fully automatic" mode:
+Be aware that Bionic can't detect every change that can affect your code's
+behavior. Bionic will inspect the bytecode of each function, as well as any
+global variables and other functions it references [#bytecode_hash]_. However, it
+won’t detect changes of the following types:
+
+1. **Installed Modules**: Bionic won’t detect changes in any built-in modules or
```

Comment:
I guess this should say "Built-In or Installed Modules". Also, "built-in modules or installed modules" should probably be "built-in or installed modules or functions".